app/views/taxons/_page_header.html.erb

```diff
@@ -1,17 +1,16 @@
 <div class="grid-row">
   <div class="column-two-thirds">
     <div class="page-header">
+      <span  role=”link” tabindex=”0″ class="sub-heading">
```

Comment:
This is why I closed it and didn't want it reviewed.  I find the idea of putting out a pull request specifically for it not to be reviewed, really weird.  I don't even know if I am keeping this.  Which I think is a waste of your time as a reviewer as well.  Thanks for pointing it out though.